src/utils/curry.js

```diff
@@ -0,0 +1,4 @@
+const curry = (f, arr = []) => (...args) =>
```

Comment:
Warning: This curry is a little naive, because it does not handle optional params! This is really for demo more than prod. Prefer ramda or lodash curry for prod.

Reply:
Replaced with Ramda